src/ripple/basics/BasicConfig.h

```diff
@@ -76,6 +76,35 @@ class Section
         return values_;
     }
 
+    /**
+     * Set the legacy value for this section.
+     */ 
+    void
+    legacy(std::string value)
+    {
+        if (lines_.empty())
+            lines_.emplace_back(std::move(value));
+        else
+            lines_[0] = std::move(value);
```

Comment:
I should definitely assert if there is more than one line, as the value could not be retrieved. I think it's OK to overwrite an existing one-line section with a new legacy value tho, and that case should not assert.